js/Makefile

```diff
@@ -201,16 +223,16 @@ android.log:
 .PHONY: android.emulator
 android.emulator:
 	$(call check-program, adb emulator)
-	$(eval ANDROID_EMULATOR ?= `emulator -list-avds | head -n1`)
+	$(eval ANDROID_EMULATOR ?= $(shell emulator -list-avds | head -n1))
```

Comment:
I'm not 100% sure, but with my current understanding on how makefile works, I think this will be called even if we manually set the var at call time

Reply:
This rule aims to easily launch the first Android emulator found in the list of those available on the machine (basically: list the AVDs, launch the first one).
So the var ANDROID_EMULATOR is not intended to be set by the user of the Makefile, so I'll pass it in lowercase to make it clearer.